vsintegration/tests/UnitTests/LegacyLanguageService/Tests.LanguageService.Completion.fs

```diff
@@ -6239,7 +6239,7 @@ let rec f l =
             marker = "(*MarkerMethodInType*)",
             list = ["PrivateMethod"])       
 
-    [<Test>]
+//    [<Test>]
```

Comment:
Best to use `Disabled` or `Skip` or whatever this test quite uses

Reply:
Sorry, I should've tried to use some attribute indeed. I looked at the code around and used the same approach.